client/my-sites/email/email-management/home/email-plan.jsx

```diff
@@ -197,9 +197,10 @@ class EmailPlan extends React.Component {
 		}
 
 		const managePurchaseUrl = getManagePurchaseUrlFor( selectedSite.slug, purchase.id );
+
 		return (
 			<VerticalNavItem path={ managePurchaseUrl }>
-				{ translate( 'Billing and payment settings' ) }
+				{ translate( 'View billing and payment settings' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.